### DISCUSSION TITLE
Beta Update v3.0.10

### DISCUSSION THREAD

**Discussion by IceHacks** [IceHacks/SurvivCheatInjector#826] [Announcements]
Hello everyone!

You may already know that the **AutoUpdate** is broken. That is why I am posting this release here. This release fixes the **AutoUpdate** and adds features such as **BulletDodge**. You should see some **improvement** in your gameplay using this version.

Thanks and Happy New Year.

###### Remember to download the file appropriate for your OS. _Chrome OS not officially supported._

<hr /><em>This discussion was created from the release <a href='https://github.com/IceHacks/SurvivCheatInjector/releases/tag/v3.0.10'>Beta Update v3.0.10</a>.</em>

**Comment by vubaduy** [comment]
I have extracted the zip file and tried to load it into my browser extension but it still doesn't work, can anyone help me about this?

**Reply by IceHacks** [reply]
It is not an extension.

**Reply by travisss25** [reply]
So how do I download it?

**Reply by viridianknight7** [reply]
https://docs.google.com/document/d/1ytRlUEKVLyj35PSHWNJCyqs-VgCeiqzQlt8OW20qATs/edit?usp=sharing

**Comment by iBLiSSIN** [comment]
Fix grenades pls

**Reply by IceHacks** [reply]
What is broken with them

**Reply by sealvessel** [reply]
They should not autofire.

**Reply by bdywbc** [reply]
![image](https://user-images.githubusercontent.com/99698341/153970975-08512462-3652-457f-8890-ec516387a8b2.png)

**Reply by bdywbc** [reply]
i can't phay ???

**Reply by viridianknight7** [reply]
lterally shut up you bozo

**Reply by bdywbc** [reply]
help me plz

**Reply by bdywbc** [reply]
> lterally shut up you bozo

can you play it

**Reply by viridianknight7** [reply]
https://drive.google.com/file/d/1HN11_qSkpMEe-lN-C3E3bJwESu3fuTYU/view?usp=sharing

**Comment by TylerWhat** [comment]
lol

**Comment by shittet** [comment]
plz update aimbot and do inverted aim so no ban easier and for pros please

**Reply by IceHacks** [reply]
Haven't you asked this question already? It is called AimOnClick.

**Reply by sealvessel** [reply]
Aimonclick is quite poor with snipers, so I use autoshoot instead.

**Comment by travisss25** [comment]
How do I download the hacks on my PC?

**Reply by viridianknight7** [reply]
https://docs.google.com/document/d/1ytRlUEKVLyj35PSHWNJCyqs-VgCeiqzQlt8OW20qATs/edit?usp=sharing

Here. Now shut up.

**Comment by shittet** [comment]
thanks

**Comment by sealvessel** [comment]
Does not launch on macos
<img width="1307" alt="image" src="https://user-images.githubusercontent.com/66351949/149302155-16fb9da0-0db7-4acf-884f-460dddf77fe1.png">
3.0.6 works, no green screen
@IceHacks

**Reply by renedbyLG** [reply]
where is 3.06

**Comment by happyface999** [comment]
yo can i get any hacks on chromebook???

**Reply by sealvessel** [reply]
no

**Comment by GInetg** [comment]
почему когда захожу в игру у меня всё зелёное?

**Comment by lmaololp** [comment]
my screen is green ;( i see the spliter bullet perk and item picked up, i can't play or leave help

**Reply by viridianknight7** [reply]
Just cope I guess?

**Reply by lmaololp** [reply]
no i think the hack having big problem i can't play anymore i try delete and download it back but still not work help ;(

**Comment by TFAGaming** [comment]
Doesn't launch on Windows (V3.0.11)
![2022-01-29 19_49_19-IceHacks - Surviv](https://user-images.githubusercontent.com/92172698/151674066-223f84e2-f2ec-4259-9013-366e83cc73b4.png)

**Comment by fuegoxxx** [comment]
hello can you make a hack for this game pokexgames see if they are pro

**Comment by fuegoxxx** [comment]
hello can you make a hack for this game pokexgames see if they are pro

**Comment by fuegoxxx** [comment]
the hack is for 64bit and i have 32bit

**Comment by fuegoxxx** [comment]
sci-3.0.9.exe did not work for me because my operating system is 32bit
and the program is 64bit

[image: Captura.PNG]
could do for 32bits

**Reply by viridianknight7** [reply]
Ooop sorry about that, guess you can't play with hacks then. I dont use mac or 32bit.

**Comment by renedbyLG** [comment]
Heyy i always get the green green on the hack or its not loading

**Reply by viridianknight7** [reply]
https://drive.google.com/file/d/1HN11_qSkpMEe-lN-C3E3bJwESu3fuTYU/view?usp=sharing

**Reply by viridianknight7** [reply]
Use this and uninstall the one you have right now

**Reply by renedbyLG** [reply]
it just loads forever

**Reply by renedbyLG** [reply]
i have a 64bit to
